deploy/README.md

```diff
@@ -40,14 +40,13 @@ domain name: `https://PROJECT_ID.appspot.com`
 
 ### Deploy the API Server
 
-* Make sure that `api/dataset_config/current` contains your dataset config
```

Comment:
This bullet point and subpoints could be reduced down to:

* Set DATASET_CONFIG_DIR in `app.yaml`

With an optional subpoint for copying platinum genomes config for test data.

Reply:
Done